redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -493,7 +702,16 @@ impl BackendCtxInterfaceUninitialised for V8Backend {
 
     fn initialize(
         self: Box<Self>,
+        logger: &'static dyn log::Log,
     ) -> Result<Box<dyn BackendCtxInterfaceInitialised>, GearsApiError> {
+        unsafe {
```

Comment:
Please explain why this must be unsafe?

Reply:
Can you explain why we need this logger from the first place? I thought the logger should just work and that inherit it from RedisGears core? Its not the case?

Reply:
Can we comment it in the code so we will have it documented?